crates/re_viewport/src/viewport.rs

```diff
@@ -273,6 +275,7 @@ impl<'a, 'b> Viewport<'a, 'b> {
             let mut tab_viewer = TabViewer {
                 viewport_state: state,
                 ctx,
+                viewport_blueprint: blueprint,
                 space_views: &blueprint.space_views,
```

Comment:
maybe drop `space_views` if we pass in the entire blueprint anyways

Reply:
good catch